proto/decentraland/social_service/v2/social_service_v2.proto

```diff
@@ -425,6 +425,78 @@ message GetIncomingPrivateVoiceChatRequestResponse {
   }
 }
 
+// Community Voice Chat messages
+
+// Community voice chat credentials - specific type for community chats
+message CommunityVoiceChatCredentials {
+  string connection_url = 1;
+}
+
+// Starting a community voice chat
+message StartCommunityVoiceChatPayload {
+  string community_id = 1;
+}
+
+message StartCommunityVoiceChatResponse {
+  message Ok {
+    string voice_chat_id = 1;
+    CommunityVoiceChatCredentials credentials = 2;  // Moderator gets credentials immediately
+  }
+
+  oneof response {
+    Ok ok = 1;
+    InvalidRequest invalid_request = 2;
+    ForbiddenError forbidden_error = 3;
+    ConflictingError conflicting_error = 4;
+    InternalServerError internal_server_error = 5;
+  }
+}
+
+// Joining a community voice chat
+message JoinCommunityVoiceChatPayload {
+  string community_id = 1;
+}
+
+message JoinCommunityVoiceChatResponse {
+  message Ok {
+    string voice_chat_id = 1;
+    CommunityVoiceChatCredentials credentials = 2;
+  }
+
+  oneof response {
+    Ok ok = 1;
+    InvalidRequest invalid_request = 2;
+    ForbiddenError forbidden_error = 3;
+    NotFoundError not_found_error = 4;
+    ConflictingError conflicting_error = 5;
+    InternalServerError internal_server_error = 6;
+  }
+}
+
+// Getting active community voice chats for user's communities
+message GetCommunityVoiceChatsResponse {
+  message CommunityVoiceChat {
+    string voice_chat_id = 1;
+    string community_id = 2;
+    int64 created_at = 3;
+  }
+
+  message Ok {
+    repeated CommunityVoiceChat active_voice_chats = 1;
+  }
+
+  oneof response {
+    Ok ok = 1;
+    InternalServerError internal_server_error = 2;
+  }
+}
+
+// Community voice chat updates/events - only 'started' status
+message CommunityVoiceChatUpdate {
```

Comment:
I completely forgot, but, we need to cover the following case as well:
1. A member A of a community is connected
2. A moderator of a community starts a community voice chat
3. Everyone receives the update saying that there's a community voice chat that has started
4. The member A never joins the community voice chat but is still connected
5. The community voice chat ends
6. The member A must be notified that the community voice chat ended so it stops showing that the community has an active one.

The problem with this case is that we might need to either:
- Tell the E@ to poll if the voice chat is still live
- Make the comms-gatekeeper communicate with the social-service-ea to tell it that the voice chat has ended 😢

Reply:
I was thinking the E@ can react on the LV room finished event, shouldn't that be sufficient? to avoid the 'end' status in this object and also the CGK having to ask SS about the voice chat status all the time. WDYT?